projects/word-weaver/src/app/app/app.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit } from "@angular/core";
-import { marker } from "@biesbjerg/ngx-translate-extract-marker";
+import { marker as _ } from "@colsen1991/ngx-translate-extract-marker";
```

Comment:
I've seen this syntax recommended somewhere, and I guess we can use it, but I don't like it because it obfuscates stuff. I would prefer continuing to import this as `marker` and spell out marker everywhere we use it.

Reply:
I agree, but `marker` no longer seemed to be recognized - I had to use `_` by the new `ngx-translate-extract` library